requirements.txt

```diff
@@ -1,21 +1,42 @@
-boto3==1.7.49
-botocore==1.10.49
-click==6.7
+-i https://pypi.python.org/simple
+argparse==1.4.0
+boto3==1.9.16
+botocore==1.12.16
+certifi==2018.8.24
+chardet==3.0.4
+click==7.0; python_version != '3.1.*'
+configparser==3.5.0
+databricks-cli==0.8.2
 docutils==0.14
-Flask==1.0.2
+flask==1.0.2
+gitdb2==2.0.4
+gitpython==2.1.11
+gunicorn==19.9.0; python_version != '3.1.*'
+idna==2.7
 itsdangerous==0.24
-Jinja2==2.10
+jinja2==2.10
 jmespath==0.9.3
-MarkupSafe==1.0
-numpy==1.14.5
-pandas==0.23.1
+markupsafe==1.0
+mleap==0.8.1
+mlflow==0.7.0
+nose-exclude==0.5.0
+nose==1.3.7
+numpy==1.15.2
+pandas==0.23.4
+protobuf==3.6.1; python_version != '3.1.*'
 pyarrow==0.9.0
```

Comment:
This version differs to the one in Pipfile....

Reply:
`requirements.txt` is there, because s2i still uses it. I havent switched to Pipfile for it, because of some problems I can't remember. Ah, maybe because of issue #16